src/pages/explore.jsx

```diff
@@ -139,43 +157,35 @@ export default function Explore() {
-                        "url(/src/assets/images/icons/down-arro.png)",
-                    }}
-                    className="bg-no-repeat bg-cover w-4 h-3 inline-block ml-2"
-                  ></span>
+                  style={{
+                    backgroundImage:
+                      "url(/src/assets/images/icons/down-arro.png)",
+                  }}
+                  className="bg-no-repeat bg-cover w-4 h-3 inline-block ml-2"
+                ></span>
               </div>
               <div className="pb-6">
-                <h2 className="text-2xl font-medium text-[#313339] mb-4">How does Switch work?</h2>
+                <h2 className="text-2xl font-medium text-[#313339] mb-4">
+                  How does Switch work?
+                </h2>
                 <div className="bg-white border border-[#2B2D331F] border-b-[3px] p-3 rounded">
-                    <div className="flex items-center justify-center gap-3 mb-4">
-                        <div className="bg-[#357065] rounded-tl rounded-bl py-4 px-3.5">
-                        <span className="text-base font-extrabold leading-5 text-white">1</span>
-                        </div>
-                        <div>
-                        <p className="text-sm font-semibold leading-[22px] text-[#313339] mb-1.5">Select where to earn 2% cashback</p>
-                        <p className="text-xs font-normal leading-[18px] text-[#767676]">Choose either shopping, dining, or travel for 2% cashback</p>
-                        </div>
-                    </div>
-                    <div className="flex items-center justify-center gap-3 mb-4">
-                        <div className="bg-[#357065] rounded-tl rounded-bl py-4 px-3.5">
-                        <span className="text-base font-extrabold leading-5 text-white">2</span>
-                        </div>
-                        <div>
-                        <p className="text-sm font-semibold leading-[22px] text-[#313339] mb-1.5">Pay with UPI or credit card</p>
-                        <p className="text-xs font-normal leading-[18px] text-[#767676]">Earn 2% cashback on your selected category and 0.4% on all other spends</p>
-                        </div>
-                    </div>
-                    <div className="flex items-center justify-center gap-3 mb-4">
-                        <div className="bg-[#357065] rounded-tl rounded-bl py-4 px-3.5">
-                        <span className="text-base font-extrabold leading-5 text-white">3</span>
-                        </div>
-                        <div>
-                        <p className="text-sm font-semibold leading-[22px] text-[#313339] mb-1.5">Switch with 1-tap</p>
-                        <p className="text-xs font-normal leading-[18px] text-[#767676]">You can switch to a different category every 3 billing cycles</p>
-                        </div>
+                  {features.map((item, index) => (
+                    <div className="flex gap-3 mb-4">
```

Comment:
when we are adding margin mb-4 so its taking margin for all the cards we dont want margin bottom from the last card. please fix it.